modules/ocf_mirrors/files/project/kde/sync-archive

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -eu
+/usr/local/bin/rsync-no-vanished -az --delete --safe-links \
+	rsync.kde.org::kdeftp /opt/mirrors/ftp/kde
+/usr/local/bin/rsync-no-vanished -az --delete --safe-links \
+	rsync.kde.org::applicationdata /opt/mirrors/ftp/kde-applicationdata
```

Comment:
I think it might make sense to mirror from `rsync://rsync.kde.org/alldata` instead, so that you then mirror both in one command essentially. Then, when you sign up as a mirror, the `kde` stuff is under `ftp/`, and `applicationdata` is under `files/`, which will clean up the mirror root directory too. Also, you wouldn't have to resync, since you'd just move the directories to the correct place and sync once again. You can do `rsync --list-only rsync://rsync.kde.org/alldata/` to list the files available to sync.